Schema.md

```diff
@@ -8,21 +8,32 @@ of `reg` the UnixFSv2 implementation is imported from the
 [`unixfsv2`](https://github.com/ipld/js-unixfsv2) module.
 
 ```sh
-type PackageV1 struct {
+type Module struct {
   file &File
   deps PackageMap
+  builtins optional [String]
+}
+type Exports struct {
```

Comment:
exports can also be an object with properties (like a dot, for main) or a string, or an array of those.

Reply:
https://npmcdn.com/es-get-iterator@1.0.1/package.json may be helpful as a valid example

Reply:
this data-structure is per-file (or per module in native ESM terms), so if there are exports for multiple endpoints they would get multiple data-structures, each with their own `Exports` struct.

Reply:
to be clear, this export map swaps out the *file* returned for each native ESM *module*. if a package.json declares multiple endpoints then each of those files will get it’s own export map in order to branch on a file-by-file basis.

Reply:
Gotcha, sorry for my misunderstanding.

There could, however, be multiple potential files for each specifier when "exports" is used.

Reply:
@ljharb is that due to the fallback logic?

Reply:
The fallback logic, and the upcoming conditional exports, will mean that different node versions *and* different environments *and* different module systems could all point to different files. They'll all be statically determinable, but there'll be 0, 1, or N files resulting from that.

Reply:
hrm... i’m fine with holding off on supporting the full spectrum since there’s other limitations that `reg` already imposes on what you can and cannot do.